DKChainableAnimationKit/Classes/DKChainableAnimationKit.swift

```diff
@@ -165,7 +165,7 @@ public class DKChainableAnimationKit {
 
     private func animateChainLink() {
         self.makeAnchor(0.5, 0.5)
-        if let animationCluster = self.animationCalculationActions.first {
+        if let animationCluster = self.animationCalculationActions.first, let _ = self.view {
             for action in animationCluster {
```

Comment:
it makes sure the view is not nil before calling `action(view)` I had a crash as well without this check.
I can change this to `guard view != nil else { return }` at the top of the method if you prefer